shared/common-adapters/safe-area-view.native.js

```diff
@@ -1,27 +1,61 @@
 // @flow
 import * as React from 'react'
 import {SafeAreaView, View, StatusBar} from 'react-native'
+import type {Props, SafeAreaViewTopBottomProps} from 'react-native'
 import * as Styles from '../styles'
 
 // Android doesn't have an implementation for SafeAreaView, so add a special case for handling the top of the screen
-type SafeAreaViewTopProps = {
-  style?: ?Styles.StylesCrossPlatform,
-  children?: React.Node,
-}
-export const SafeAreaViewTop = ({style, children}: SafeAreaViewTopProps) =>
+export const SafeAreaViewTop = ({style, children}: Props) =>
   Styles.isAndroid ? (
     <View style={Styles.collapseStyles([styles.androidTopSafeArea, style])}>{children}</View>
   ) : (
     <SafeAreaView style={Styles.collapseStyles([styles.topSafeArea, style])}>{children}</SafeAreaView>
   )
 
+export const SafeAreaViewTopBottom = (props: SafeAreaViewTopBottomProps) => (
```

Comment:
spoke irl , don't think this is necesasry. you can just use a SafeAreaView on th ebottom of the screen w/ whatever color you want

Reply:
@chrisnojima I pushed a different approach, ptal